google-cloud-spanner-hibernate-performance-testing/src/main/java/com/google/cloud/spanner/hibernate/ClientLibraryOperations.java

```diff
@@ -72,9 +72,10 @@ public ClientLibraryOperations() {
   }
 
   /**
-   * Creates a database.
+   * Resets the test Spanner database.
    */
-  public void createTestDatabase() {
+  public void resetTestDatabase() {
+    databaseAdminClient.dropDatabase(INSTANCE_NAME, DATABASE_NAME);
```

Comment:
My recollection is that dropping a nonexistent database results in exception log clutter. If it still does that, might want to check if database exists first.

Reply:
Done; I decided to just surround this in a try/catch and suppress the error if it is a table-not-found error.